python/sqlcommenter-python/tests/sqlalchemy/tests.py

```diff
@@ -126,3 +126,38 @@ def test_route_disabled(self, get_info):
             "SELECT 1; /*controller='c',framework='flask'*/",
             with_route=False,
         )
+
+
+class FastAPITests(SQLAlchemyTestCase):
+    fastapi_info = {
+        'framework': 'fastapi',
+        'controller': 'c',
+        'route': '/',
+    }
+
+    @mock.patch('google.cloud.sqlcommenter.sqlalchemy.executor.get_fastapi_info', return_value=fastapi_info)
+    def test_all_data(self, get_info):
+        self.assertSQL(
+            "SELECT 1; /*controller='c',framework='fastapi',route='/'*/",
```

Comment:
Sorry to miss this one. 
Although, this change looks good, wanted to give you an FYI on an existing bug in SQLAlchemy and psycopg2 sqlcommenter:

This query:
```
SELECT 1; /*controller='c',framework='fastapi',route='/'*/
```

In general a `;` ends a SQL query. So, any comment added after that is not really part of that query. So, even with this change, my guess is SQL comments would not work as expected as DB will assume there are 2 queries:
1. SELECT 1;
2. /\*controller='c',framework='fastapi',route='/'\*/

I have added one small [PR](https://github.com/google/sqlcommenter/pull/112) for psycopg2 as of now (Waiting for review) and will create another for SQLAlchemy if the approach doesn't have major concern.